src/lib/cfa/nm_connection.rb

```diff
@@ -31,15 +35,26 @@ class NmConnection < BaseModel
     KNOWN_SECTIONS = [
       "bridge", "connection", "ethernet", "ipv4", "ipv6", "vlan", "wifi", "wifi_security"
     ].freeze
+    SYSTEM_CONNECTIONS_DIR = "/etc/NetworkManager/system-connections".freeze
 
+    class << self
+      # Returns all connection definition files
+      #
+      # @return [Array<NmConnection>]
+      def all
+        directory = File.join(Yast::Installation.destdir, SYSTEM_CONNECTIONS_DIR)
+        files = Dir[File.join(directory, "*.nmconnection")].to_a
```

Comment:
If I'm not wrong NetworkManager reads any ini file under the system-connections directory.

Reply:
You are right. Fixed :-)